generator/templates/tupleserializer.gotmpl

```diff
@@ -62,6 +62,10 @@ func ({{.ReceiverName}} {{ pascalize .Name }}) MarshalJSON() ([]byte, error) {
 }
 {{ end }}
 
+{{- define "dedupe" }}{{/* add a prefix if name conflicting with a local variable */}}
```

Comment:
this should probably move to a function instead a template? Or even be generated an part of the GenXxx struct that describes this data

Reply:
Yeah I pondered between these 3... The dedupe is very "local", so eventually I put it close to where it is used...